docs/docsite/rst/scenario_guides/guide_docker.rst

```diff
@@ -15,13 +15,19 @@ Most of the modules and plugins in community.docker require the `Docker SDK for
 
 Note that plugins (inventory plugins and connection plugins) are always executed in the context of Ansible itself. If you use a plugin that requires the Docker SDK for Python, you need to install it on the machine running ``ansible`` or ``ansible-playbook`` and for the same Python interpreter used by Ansible. To see which Python is used, run ``ansible --version``.
 
-You can install the Docker SDK for Python for Python 2.7 or Python 3 as follows:
+You can install the Docker SDK for Python for Python 3.6 or later as follows:
 
 .. code-block:: bash
 
     $ pip install docker
 
-For Python 2.6, you need a version before 2.0. For these versions, the SDK was called ``docker-py``, so you need to install it as follows:
+For Python 2.7, you need to use a version between 2.0.0 and 4.4.4 since removed support for Python 2.7 on 5.0.0. You can install the specific version of the Docker SDK for Python as follows:
```

Comment:
```suggestion
For Python 2.7, you need to use a version between 2.0.0 and 4.4.4 since the Python package for Docker removed support for Python 2.7 on 5.0.0. You can install the specific version of the Docker SDK for Python as follows:
```